src/main/scala/com/lightbend/coursegentools/Helpers.scala

```diff
@@ -363,26 +363,29 @@ object Helpers {
 
   def createMasterBuildFile(exercises: Seq[String],
                             masterRepo: File,
-                            multiJVM: Boolean)(implicit config: MasterSettings): Unit = {
+                            multiJVM: Boolean,
+                            isADottyProject: Boolean)(implicit config: MasterSettings): Unit = {
 
     val targetFolder = new File(masterRepo, config.relativeSourceFolder)
 
     val exercisesBackTicked = exercises.map(exrc => s"`$exrc`")
 
+    val (dottyPreamble, setScalaVersion) = scalaDottyPreamble(isADottyProject)
```

Comment:
Shouldn't the `setScalaVersion` go into `commonSettings` instead?
But I guess this hack is OK.

Reply:
There's some interference with the dotty plugin that led me to use this strange implementation...
Once Scala 3 is released, this will no longer be necessary.